frontend/src/pages/RollingpaperPage/components/Like.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useState } from "react";
+import styled from "@emotion/styled";
+
+import EmptyHeart from "@/assets/icons/bx-heart-empty.svg";
+import FilledHeart from "@/assets/icons/bx-heart-fill.svg";
+
+import useUpdateLike from "@/pages/RollingpaperPage/hooks/useUpdateLike";
+import useDeleteLiked from "@/pages/RollingpaperPage/hooks/useDeleteLiked";
+
+import { Message } from "@/types";
+
+interface LikeProps extends Pick<Message, "id" | "likes" | "liked"> {}
+
+const Like = ({ id, likes, liked }: LikeProps) => {
+  const [count, setCount] = useState(likes);
+  const [isLiked, setIsLiked] = useState(liked);
+
+  const { updateLike } = useUpdateLike();
+  const { deleteLiked } = useDeleteLiked();
+
+  const handleLikeToggle = () => {
+    if (isLiked) {
+      deleteLiked(id, {
+        onSuccess: ({ liked, likes }) => {
+          setCount(likes);
+          setIsLiked(liked);
+        },
+      });
+
+      return;
+    }
+
+    updateLike(id, {
+      onSuccess: ({ liked, likes }) => {
+        setCount(likes);
+        setIsLiked(liked);
+      },
+    });
+  };
+
+  return (
+    <StyledLike>
+      {isLiked ? (
+        <StyledFilledHeart onClick={handleLikeToggle}>
+          <FilledHeart />
+        </StyledFilledHeart>
+      ) : (
+        <StyledEmptyHeart onClick={handleLikeToggle}>
+          <EmptyHeart />
+        </StyledEmptyHeart>
+      )}
+      <StyledLikeCount>{count}</StyledLikeCount>
+    </StyledLike>
+  );
+};
+
+const StyledLike = styled.div`
+  display: flex;
+  align-items: flex-start;
+  align-self: flex-end;
+  gap: 8px;
+
+  height: 20px;
+
+  color: ${({ theme }) => theme.colors.GRAY_700};
+`;
+
+const StyledFilledHeart = styled.div`
+  svg {
+    fill: ${({ theme }) => theme.colors.RED_300};
```

Comment:
> 좋아요를 채우면 기존의 빨간색 메시지 종이와 색상이 동일합니다. 메시지 색상을 변경할지 고민입니다.

하트 채우기 색상을 더 진하게 바꾸면 잘 보이네요! 저는 400으로 바꿔도 좋습니다~
```suggestion
    fill: ${({ theme }) => theme.colors.RED_400};
`;
```